Dockerfile

```diff
@@ -76,6 +71,14 @@ RUN yarn run build
 # Delete any sourcemaps which may have been generated to avoid creating a large artifact.
 RUN find /calypso/build /calypso/public -name "*.*.map" -exec rm {} \;
 
+###################
+# A cache-only update can be generated with "docker build --target update-base-cache"
+FROM ${base_image} as update-base-cache
```

Comment:
In practice, this image isn't triggered when building the file, just because the final target (`app`) doesn't reference this image. However, we can take advantage of `--target` to tell `docker build` to just run this section. Given that each layer has been cached by docker, a separate `docker build` command which targets the update cache image should take advantage of the previously updated .cache files. (This happened in `yarn build` in the image above.)